pg-index-health/src/test/java/io/github/mfvanek/pg/support/statements/CreateNotSuitableIndexForForeignKeyStatement.java

```diff
@@ -10,19 +10,14 @@
 
 package io.github.mfvanek.pg.support.statements;
 
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.util.List;
 import javax.annotation.Nonnull;
 
 public class CreateNotSuitableIndexForForeignKeyStatement extends AbstractDbStatement {
 
-    public CreateNotSuitableIndexForForeignKeyStatement(@Nonnull final String schemaName) {
-        super(schemaName);
-    }
-
     @Override
-    public void execute(@Nonnull final Statement statement) throws SQLException {
-        statement.execute(String.format("create index if not exists " +
+    protected List<String> getSqlToExecute(@Nonnull final String schemaName) {
```

Comment:
Now we can to go further and get rid of `String.format` using kind of templating `{schemaName}`

```
    protected List<String> getSqlToExecute() {
        return List.of("create index if not exists " +
            "i_accounts_account_number_client_id on {schemaName}.accounts (account_number, client_id)");
    }

```

and process the template like
```
// AbstractDbStatement.java

        final String schemaName = getSchemaName();
        for (final String sql : getSqlToExecute(schemaName)) {
            statement.execute(sql.replace("{schemaName}", schemaName));
        }
```